app/controllers/link_controller.rb

```diff
@@ -10,7 +10,7 @@
 class LinkController < ApplicationController
   bing_search_api = BingService::BingWebSearchApi.new
 
-  get '/category_links/:category_id' do
+  get '/category/:category_id/links' do
```

Comment:
I like the new restful routes. 
Just a small issue: I'd have expected `category` to be plural. Check out the link that I shared on RESTful routes.